app/src/main/java/org/mozilla/fenix/library/bookmarks/edit/EditBookmarkFragment.kt

```diff
@@ -120,6 +121,12 @@ class EditBookmarkFragment : Fragment() {
         updateBookmarkFromObservableInput()
     }
 
+    override fun onPause() {
+        super.onPause()
+        bookmark_name_edit.hideKeyboard()
+        bookmark_url_edit.hideKeyboard()
```

Comment:
Personal story: I dislike seeing snake case when the rest of the code is in camel case.

Other than that, this is a reasonable way of fixing the problem. Well done.